week-2/errors/1.js

```diff
@@ -3,11 +3,26 @@
 // Why will an error occur when this program runs?
 // Play computer with the example to work out what is going on
 
+//This will not work because in line 7 the variable decimal number declaration came after the function. It should come first then the function declaration.
+//Also we need to call the function in the console log for it to work.
+
+// function convertToPercentage(decimalNumber) {
+//   const decimalNumber = 0.5;
+//   const percentage = `${decimalNumber * 100}%`;
+
+//   return percentage;
+// }
+
+// console.log(decimalNumber);
+
+//I have corrected the code below
+
 function convertToPercentage(decimalNumber) {
-  const decimalNumber = 0.5;
+ 
   const percentage = `${decimalNumber * 100}%`;
 
   return percentage;
 }
 
-console.log(decimalNumber);
+
+console.log(convertToPercentage(0.7));
```

Comment:
What you did here is correct, however, you need to declare decimalNumber as a var and assign 0.7 to it, and then pass the "decimalNumber" into the function  convertToPercentage instead of passing 0.7 straight.